hms-app/src/components/PitchTimer.tsx

```diff
@@ -110,6 +157,12 @@ function PitchTimer(time: PitchTimerProps) {
 
   return (
     <div className={classes.outerBox}>
+      <Progress
+        className={classes.progress}
+        value={progress}
+        color={isCompleted() ? 'red' : 'gray'}
```

Comment:
using colors defined outside this file might be better (allow for easier customization and ensures we use the same colors for similar things)

Reply:
these are MantineColor identifiers so are basically keys for MantineColors that are defined in the theme